### DISCUSSION TITLE
Feasibility of an iPad port?

### DISCUSSION THREAD

**Discussion by Hikiru** [godotengine/godot-proposals#8542] [Platforms]
I just watched a video where someone attempted porting godot to iOS, and it seemed the main limitation was that they couldn’t switch between the project manager, editor, and game player without making 3 separate apps, which seemed a bit silly to do. How hard would it be to do something similar to the web editor, using 3 separate tabs for each? Would this be allowed on the App Store? Even if it wouldn’t, I would still like to see it done so I could side load it. I don’t have a laptop, so being able to edit and create projects on my iPad would be great. Even if it’s just a straight port of the PC UI, I don’t mind since I have the Magic Keyboard.

**Comment by AThousandShips** [comment]
See:
* https://github.com/godotengine/godot-proposals/issues/1061

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/1061.

**Comment by bruvzg** [comment]
See https://github.com/godotengine/godot-proposals/issues/8151, this is essentially the only thing preventing iOS port.
